dataprofiler/profilers/column_profile_compilers.py

```diff
@@ -47,21 +51,12 @@ def _create_profile(self, df_series, options=None):
         if len(self._profilers) == 0:
             return
 
-        # convert all the values to string
-        df_series = df_series.apply(str)
-
-        multiprocess_flag = True
         selected_col_profiles = None
         if options and isinstance(options, StructuredOptions):
             selected_col_profiles = options.enabled_columns
-            multiprocess_flag = options.multiprocess.is_enabled
-
-
-        single_process_list = []
-        multi_process_dict = {}
-        if multiprocess_flag: # only create pool when necessary
-            pool = mp.Pool(len(self._profilers))
+            self.multiprocess_flag = options.multiprocess.is_enabled
```

Comment:
any reason this is set here? I think would make the most sense int `__init__` as that's where options previously were set and it doesn't impact the profile creation.

Reply:
will be fixed in future PR anyways so ignoring